app/src/renderer/vuex/modules/node.js

```diff
@@ -43,13 +43,14 @@ export default function ({ node }) {
         dispatch('reconnected')
       }
     },
-    nodeSubscribe ({commit, dispatch}) {
+    async nodeSubscribe ({commit, dispatch}) {
       if (state.stopConnecting) return
 
       // the rpc socket can be closed before we can even attach a listener
       // so we remember if the connection is open
       // we handle the reconnection here so we can attach all these listeners on reconnect
       if (!node.rpcOpen) {
+        await sleep(500)
```

Comment:
awesome

Reply:
This was needed so the process isn't blocked while continuously  trying to reconnect.